### PR TITLE
Add most popular libraries to InAppExclude

### PR DESCRIPTION
**Pull request by bruno-garcia**
Frames marked `InApp=true` are considered when grouping issues in Sentry.
To help improve grouping, excluding a few more namespaces that are known to be external packages

### CODE REVIEW

File: src/Sentry/SentryOptions.cs [hunk 1]
```diff
@@ -401,6 +401,30 @@ public SentryOptions()
...
+                    "Polly",
+                    "Swashbuckle",
+                    "Fluentassertions",
+                    "Autofac",
+                    "Stackexchange.Redis",
+                    "Dapper",
+                    "RestSharp",
+                    "SkiaSharp",
+                    "IdentityModel",
+                    "SqlitePclRaw",
```
Comment (bruno-garcia) [inline src/Sentry/SentryOptions.cs:418]:
```suggestion
                    "SqlitePclRaw",
                    "Xamarin",
```

**Comment by bruno-garcia** [comment]
@Tyrrrz did the tests flakey up now? Seem to break at random

**Comment by Tyrrrz** [comment]
@bruno-garcia yes, AppVeyor did:

image

Not a test issue.

File: src/Sentry/SentryOptions.cs [hunk 2]
```diff
@@ -401,6 +401,31 @@ public SentryOptions()
...
                     "Giraffe.",
                     "NLog",
+                    "Npgsql",
+                    "RabbitMQ",
+                    "Hangfire",
+                    "IdentityServer4",
+                    "AWSSDK",
+                    "Polly",
+                    "Swashbuckle",
+                    "Fluentassertions",
```
Comment (Tyrrrz) [inline src/Sentry/SentryOptions.cs:411]: Why would someone use Fluentassertions in an application? Should it be FluentValidation?

**Review by Tyrrrz — APPROVED** [review]

**Comment by Tyrrrz** [comment]
Is the purpose of this feature something like "show only my code"? Maybe in that case it's possible to implement some smarter heuristic in the future, instead of listing every single library in existence?

**Comment by Tyrrrz** [comment]
Ok. one test did appear to fail, didn't notice that at first.

image

Judging from the stack trace, the error comes from the constructor of `Sentry.Internal.Hub`. Which shouldn't be called in this test because DSN is empty:

image

My guess is that the tests might be running in parallel and there's a race condition where some other test may set DSN environment variable, which gets read in this test in `var dsn = options.Dsn ?? DsnLocator.FindDsnStringOrDisable();`, resulting in the wrong behavior.

File: src/Sentry/SentryOptions.cs (hunk 2 above)
Comment (bruno-garcia) [reply src/Sentry/SentryOptions.cs:411]: yes

**Comment by bruno-garcia** [comment]
@Tyrrrz yeah seems something got flakey

> FluentValidation

Will fix it

> yeah something with Assembly.GetEntrypoint().Namepsace of sorts but it comes with other down sides.

Would be worth a try if you're willing to give it a shot.

It's not *just* for the "just my code" feature, but also affects grouping. We only consider "InApp" frames when grouping issues, so those async await (MoveNext) frames in between shouldnt play a role, otherwise it gets badly grouped due to often differnt generated code frames in the middle

File: src/Sentry/SentryOptions.cs (hunk 2 above)
Comment (bruno-garcia) [inline src/Sentry/SentryOptions.cs:411]:
```suggestion
                    "FluentValidation",
```

**Pull request merged**